TODO/a-fairer-vue-of-react-comparing-react-to-vue-for-dynamic-tabular-data-part-2.md

```diff
@@ -1,83 +1,84 @@
 >* 原文链接 : ["A fairer Vue of React" - Comparing React to Vue for dynamic tabular data, part 2.](https://engineering.footballradar.com/a-fairer-vue-of-react-comparing-react-to-vue-for-dynamic-tabular-data-part-2/)
 * 原文作者 : [Max Willmott](https://engineering.footballradar.com/author/max-willmott/)
 * 译文出自 : [掘金翻译计划](https://github.com/xitu/gold-miner)
-* 译者 : 
+* 译者 : @wildflame
 * 校对者:
 
 
-_For part one, please visit: [https://engineering.footballradar.com/from-a-react-point-of-vue-comparing-reactjs-to-vuejs-for-dynamic-tabular-data/](https://engineering.footballradar.com/from-a-react-point-of-vue-comparing-reactjs-to-vuejs-for-dynamic-tabular-data/). Note that the results of this first post have been invalidated by some unfortunate mistakes, but this should set the context for this post._
+有关第一部分的文章，请访问 [https://engineering.footballradar.com/from-a-react-point-of-vue-comparing-reactjs-to-vuejs-for-dynamic-tabular-data/](https://engineering.footballradar.com/from-a-react-point-of-vue-comparing-reactjs-to-vuejs-for-dynamic-tabular-data/) 。注：因为我们犯了一些意料之外的错误，在第一部分的实验结果并不准确，但是，那篇文章奠定了这篇文章的背景。
 
-Monday 23rd May we posted the results of a performance experiment using React and Vue. The idea was to work out whether React or Vue would be more suitable for tabular data that updates a lot, especially when performance is critical. This is very representative of some challenges we work on at Football Radar.
+五月23日，周一，我们发布了一篇关于比较 React 和 Vue 的性能的文章，其实验数据比较了二者谁更适合处理频繁更新的列表数据，特别应用在那些对性能要求非常高的场合上谁更合适。比方说我们手头上的一个足球雷达（Football Radar）的项目。
 
-Initially we were very encouraged and surprised by the results, but a few important mistakes meant that our "findings" were not as revelatory as as we thought. Thanks to some great feedback from both the React and Vue communities - especially React core engineer Christopher Chedeau ([@vjeux](https://twitter.com/vjeux)) and Vue creator Evan You ([@youyuxi](https://twitter.com/youyuxi)) - we were quickly able to narrow down the problems with our tests. We're very grateful for the high-quality discussion that followed; I'm slightly embarrassed that these mistakes made it into the open but we've learnt a lot from the responses, so it's been a blessing in disguise.
+最初，我们对那些结果非常满意。但由于我们在重要的地方犯了一些错误，结果其实没那么理想，至少，结果不那么客观了。我们非常感谢 React 和 Vue 社区里，那些给我们提供反馈的人 —— 特别是 React 的核心工程师克里斯托弗(Christopher Chedeau) ([@vjeux](https://twitter.com/vjeux))，和 Vue 的创始人埃文(Evan You)([@youyuxi](https://twitter.com/youyuxi))—— 因为你们，我们才能快速的锁定这次测试中的出现的问题，可以说是因祸得福，因为尽管错误被公开使我感到有一些小小的尴尬，但我的确学到了很多，所以衷心的感谢你们的讨论。
```

Comment:
宝贵意见 确实更符合中文语境呢
在2016年06月13日 21:07，Jiegao Zhu 写道:

In TODO/a-fairer-vue-of-react-comparing-react-to-vue-for-dynamic-tabular-data-part-2.md:

>  
> -Initially we were very encouraged and surprised by the results, but a few important mistakes meant that our "findings" were not as revelatory as as we thought. Thanks to some great feedback from both the React and Vue communities - especially React core engineer Christopher Chedeau ([@vjeux](https://twitter.com/vjeux)) and Vue creator Evan You ([@youyuxi](https://twitter.com/youyuxi)) - we were quickly able to narrow down the problems with our tests. We're very grateful for the high-quality discussion that followed; I'm slightly embarrassed that these mistakes made it into the open but we've learnt a lot from the responses, so it's been a blessing in disguise.
> +最初，我们对那些结果非常满意。但由于我们在重要的地方犯了一些错误，结果其实没那么理想，至少，结果不那么客观了。我们非常感谢 React 和 Vue 社区里，那些给我们提供反馈的人 —— 特别是 React 的核心工程师克里斯托弗(Christopher Chedeau) ([@vjeux](https://twitter.com/vjeux))，和 Vue 的创始人埃文(Evan You)([@youyuxi](https://twitter.com/youyuxi))—— 因为你们，我们才能快速的锁定这次测试中的出现的问题，可以说是因祸得福，因为尽管错误被公开使我感到有一些小小的尴尬，但我的确学到了很多，所以衷心的感谢你们的讨论。


我们非常感谢 React 和 Vue 社区里，那些给我们提供反馈的人

-> 我们非常感谢在React和Vue社区里的宝贵意见

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.